README.md

```diff
@@ -25,3 +25,11 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+My work:
+
+I added the relation? im sorry for all the pings, i hope this is the last time!
+
+Recurrence relation: T(n)=3T(n/3)+O(n), Alright so I think i found my mistake. There is a constant factor because thats just how adding everthing in an array is. But because of the divide and conqure portion of the algorithm when doing the math we end up solving a sumation just like we did for the binary sorting exersise. When solved we get a final complexity for the recursive portion of the program being log(n) which we need to add the linear portion back to making the final recurrence relation equal to $T(n)$ = nlog(n). 
```

Comment:
You can't have an asymptotic expression in a recurrence relation. And please use the notation we've used in class to denote that a particular $T(n)$ is a member of an asymptotic complexity class.